src/mesh/http/WiFiAPClient.cpp

```diff
@@ -238,10 +238,8 @@ bool initWifi(bool forceSoftAP)
                 WiFi.softAPConfig(apIP, apIP, IPAddress(255, 255, 255, 0));
                 DEBUG_MSG("MY IP AP ADDRESS: %s\n", WiFi.softAPIP().toString().c_str());
 
-#if !CONFIG_IDF_TARGET_ESP32S3
                 // This is needed to improve performance.
-                esp_wifi_set_ps(WIFI_PS_NONE); // Disable radio power saving
```

Comment:
Are these no longer needed?

Reply:
They were removed from the ESP32-S3 target by Macroguard. I have not looked into the new API if there's a function equivalent for ESP32(vanilla) or if the new API just ditches this call, but the S3 doesn't seem to need it any more.